lib/puppet/provider/cs_clone/crm.rb

```diff
@@ -20,6 +20,8 @@
 
   mk_resource_methods
 
+  defaultfor operatingsystem: [:ubuntu]
```

Comment:
is `operatingsystem` the legacy fact?

Reply:
Ah. That could be. I don't really know where I've got that code from. 😄 

Yeah, apparently defaultfor uses a fact filter. I'll see how to use os.name for that.

Reply:
@dploeger this still needs to be updated for modern facts :)

Reply:
Whoops, sorry. I'll look into it.

Reply:
There you go.